docs/index.md

```diff
@@ -11,6 +11,7 @@ The core of this package is a set of classes for representing different OME-Zarr
 | [`Labels`][ome_zarr_models.v05.Labels]         | [`Labels`][ome_zarr_models.v04.Labels]         |
 | [`ImageLabel`][ome_zarr_models.v05.ImageLabel] | [`ImageLabel`][ome_zarr_models.v04.ImageLabel] |
 | [`Well`][ome_zarr_models.v05.Well]             | [`Well`][ome_zarr_models.v04.Well]             |
+| [`Plate`][ome_zarr_models.v05.Plate]           | [`Plate`][ome_zarr_models.v04.Plate]           |
```

Comment:
I'm going to take a look at the `pydantic-zarr` PR before this one, but this looks like an oversight/bug 🤦 - would you be up for putting this fix in it's own PR?